.moai/project/codemaps/modules.md

```
@@ -71,9 +71,10 @@
 **역할**: Ralph 의사결정 엔진  
 **기능**: `Decide()` (max_iter > perfect_gate > stagnation > human_review)
 
-### internal/harness (75 non-test 파일)
+### internal/harness (80 non-test 파일)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)